### PR TITLE
Setup dashboards for Istiod

### PR DESCRIPTION
**Pull request by howardjohn**
* Remove citadel/galley dashboard
* add a new "webhooks" section to the pilot dashbaord
* Rename pilot dashboard to "Control Plane"

I didn't add any other panels, because the only metrics we get are the
webhook ones. We don't get CSR or MCP metrics now.

### CODE REVIEW

File: manifests/istio-telemetry/grafana/dashboards/pilot-dashboard.json [hunk 1]
```diff
@@ -1586,7 +1792,7 @@
     ]
   },
   "timezone": "browser",
-  "title": "Istio Pilot Dashboard",
+  "title": "Istio Control Plane Dashboard",
```
Comment (carolynhu) [inline manifests/istio-telemetry/grafana/dashboards/pilot-dashboard.json]: If you rename this to Control Plane, do we need to update the file name to `control-plane-dashboard.json` also in the `dashboard_test.go` where has `pilot` part?

**Comment by howardjohn** [comment]
@douglas-reid mind taking a look? I want to get some docs for downloading dashboards from grafana.com but it makes sense to wait for this first

**Comment by howardjohn** [comment]
ping @istio/wg-policies-and-telemetry-maintainers

**Comment by douglas-reid** [comment]
@howardjohn i will take a look today.

**Comment by douglas-reid** [comment]
@howardjohn the functionality that moved into istiod from citadel/galley -- does that not need to be monitored at all?

**Comment by howardjohn** [comment]
@douglas-reid I would say yes, it does need to be monitored, but its currently not exporting the metrics. So we can't add them to the dashboard now, but we should add the metrics and then expose them in the dashboard in the future

**Comment by douglas-reid** [comment]
@howardjohn ok. filed https://github.com/istio/istio/issues/22508

**Review by douglas-reid — APPROVED** [review]

**Review by richardwxn — APPROVED** [review]

**Pull request merged**